.github/workflows/stripe-patrons-data.yml

```diff
@@ -57,6 +56,50 @@ jobs:
             ~/.coursier
           key: sbt
 
-      - name: Build and upload to RiffRaff
+      - name: Build
         run: |
-          sbt "project stripe-patrons-data" riffRaffUpload
+          sbt "project stripe-patrons-data" assembly
```

Comment:
I _think_ that `riffRaffUpload` is going to [run the tests automatically](url), whereas IIUC `assembly` [is not doing this](https://github.com/guardian/support-frontend/actions/runs/5542018268/jobs/10116159745). Consequently, I think we might need to explicitly add `sbt test` to this command?

Reply:
good spot!